### PR TITLE
Works and about sections

### PR DESCRIPTION
**Pull request by carreraprogrammer**
In this branch:

- I created the "Works" section with flex and grid display
- I created about section with flexbox
- I list languages, frameworks, and skills
- I customized all with CSS in accordance with the Figma parameters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
       <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
     </head>
     <body>
+
+    <!-- Here starts the header section -->
+
       <header>
         <nav class="navbar">
             <a class="logo-link"><p class="logo">Carrera<i  id=logo-dat class="fa-solid fa-circle-dot"></i>Programmer</p></a>
```
Comment (MaganaAsiati) [inline index.html:19]: - [ ] Kindly make sure your home page looks like in the Figma design with the same color, same format and same font size. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## Figma design

image


## your design
image

File: index.html [hunk 2]
```diff
@@ -30,5 +33,126 @@ <h2>I'm a Full Stack Developer</h2>
...
+            <div class="project-card card-one">
+                <h3 class="card-title">Personal Portfolio Carrera</h3>
+                <p class="card-description">A personal portfolio to show different kinds of projects. This portfolio is the connection between the developer and the final customer or recruiter.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                </ul>
+                 <a class="btn link-card-button" href="https://carreraprogrammer.github.io/carreraprogrammerportfolio/" target="_blank" rel="noopener">See Project</a>
+                
+            </div>
```
Comment (MaganaAsiati) [inline index.html]: - [ ] Kindly make sure your card looks like in the Figma design. Add margins (top, bottom, left, right) defined in the design as defined in the design and please make sure you use the right card you can even download it from Figma. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## Figma design

image
image


## your design
image
image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @carreraprogrammer 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :100: :grimacing: :grinning: :grinning: 

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional
✅ Descriptive pull request


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Kindly write your commit message in imperative tense and capitalized. use this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information about the way to write commit message :grin: :grin: :grin: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mwanawabangona — APPROVED** [review]
Hi @carreraprogrammer,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**